synapse/storage/schema/main/delta/59/12account_validity_token_used_ts_ms.sql

```diff
@@ -1,18 +0,0 @@
-/* Copyright 2020 The Matrix.org Foundation C.I.C.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Was an issue created to track the issues with these migration files?

Reply:
I've just opened https://github.com/matrix-org/synapse-dinsic/issues/103